<!-- -*- mode: xml; -*- -->
<launch>
  <!-- CozmoDriver -->
  <!--include file="$(find cozmo_driver)/launch/cozmo_driver.launch">
  </include-->
  <node pkg="cozmo" type="cozmo_driver.py" name="cozmo_driver" output="screen">
  </node>
</launch>

